delivery.yaml

```diff
@@ -41,6 +41,15 @@ pipeline:
             export PATH=$PATH:$HOME/go/bin
             cd $OPERATOR_TOP_DIR/postgres-operator
             go test ./...
+        - desc: 'Run e2e tests'
+          cmd: |
+            export PATH=$PATH:$HOME/go/bin
+            cd $OPERATOR_TOP_DIR/postgres-operator
+            echo "INFO install kubectl to test 'kind' from client side"
+            (curl -LO --silent https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl &&
```

Comment:
It always looks scary, especially in tests. There is a apt.kubernetes.io repository, maybe it's better to install it this way?